src/App.js

```diff
@@ -60,11 +73,25 @@ class App extends React.Component {
     });
   }
 
+  // Adds script for search bar
+  componentDidMount () {
+    const script = document.createElement("script");
+    script.src = "https://cse.google.com/cse.js?cx=013104617978576650762:hrpvx9uejrs";
+    script.async = true;
+    document.body.appendChild(script);
+  }
+
+  querySearch(query) {
+    document.getElementById("gsc-i-id1").value = query;
+    for (const button of document.getElementsByClassName("gsc-search-button gsc-search-button-v2")) {
```

Comment:
I'd imagine there is only one button to press, how come this loops through results?

Reply:
There is only one button with this class name. The button only has a class not an id and I don't build the button since the div in render creates everything. Is there another way to do this?

Reply:
I see, you could access `[0]` directly instead of looping if you know it's always there. Thanks for clarifying!